mirrord-cli/src/main.rs

```diff
@@ -39,11 +40,23 @@ fn extract_library(dest_dir: Option<String>) -> Result<PathBuf> {
     let library_file = env!("MIRRORD_LAYER_FILE");
     let library_path = std::path::Path::new(library_file);
 
-    let file_name = library_path.components().last().unwrap();
+    let extension = library_path
```

Comment:
Does this happen every time the CLI runs? Is it deleted at some point or do we just keep creating dylibs?

Reply:
> Does this happen every time the CLI runs?
Not every time, but randomly. 

>  Is it deleted at some point or do we just keep creating dylibs?

Previously, we were overwriting the dylib/so created in tmp, since we added a random string to it now, we are not doing so anymore. not sure what would be the right way to delete the old shared lib

Reply:
Don't worry about it for now since it's in /tmp anyway